Fabric.Terminology.API/Scripts/Install-Terminology.ps1

```diff
@@ -45,20 +45,35 @@ For ($i = 0; $i -lt $requiredFiles.Length; $i++) {
 }
 
 # Import Dos Install Utilities
-$MinimumVersion = [Version]::new(1, 0, 163)
-Import-Module -Name DosInstallUtilities -MinimumVersion $MinimumVersion -ErrorAction Stop
+$dosInstallUtilities = Get-Childitem -Path ./**/DosInstallUtilities.psm1 -Recurse
```

Comment:
reverting to old way of looking for DosInstallUtilites because powershell 4 doesn't support `Install-Module`